cylc/flow/scripts/scan.py

```diff
@@ -568,7 +585,21 @@ async def main(
     if opts.format == 'rich' and not opts.colour_blind:
         cprint(state_totals_key() + '\n')
 
-    await scanner(opts, write, scan_dir)
+    nflows = await scanner(opts, write, scan_dir)
+
+    if (
+        opts.format == "rich"
+        or opts.ping
+        or opts.source
+        or not nflows
+    ):
+        return
+
+    print(
+        "\nUse `--ping` to attempt to connect to schedulers and clean "
+        "\nup after any that were killed or did not shut down cleanly.",
+        file=stderr
+    )
```

Comment:
I think printing this message every time is a bit too obstructive.

Reply:
Well, it's only printed for non scheduler-contacting cases, and only to stderr.  And I got here via a bunch of confused-user questions (*"`cylc scan` says my workflow is running, but stuff isn't working!"*).

Reply:
> Well, it's only printed for non scheduler-contacting cases

So the default i.e. 99.9% of cases.

Reply:
I feel that if it is really this important that we add a message to every command, then it should really be the default.

But we don't want to do that in order to keep `cylc scan` fast.

#5511 possible alternative?

Reply:
This situation is particularly confusing for users when it does happen, because they have a reasonable expectation that `cylc scan` does not report stopped workflows as running workflows.

However, I'll remove the message.  I quite like your proposed #5511 solution.